### ISSUE TITLE
API call behind a company proxy

### ISSUE THREAD

**Issue by jeremyjong** [box/box-python-sdk-gen#195]
Hello,

I'm trying to transition my box-sdk code to box_sdk_gen, 
but I am facing connection problems when calling api behind a company proxy.

Previously, in box-sdk, all I had to do was:

```
from boxsdk.config import Proxy, API
Proxy.URL = 'http://your_proxy_url:port'
```

Is there any proxy support for the current repository? 
Any help would be appreciated, and I can provide any additional information needed.

Thanks,
Jeremy

**Comment by mwwoda** [comment]
We currently don't support proxies in this SDK, but it's on our roadmap. Unfortunately, I cannot give you precise timeframe for now.

**Comment by jeremyjong** [comment]
Thank you for the reply. I'll check periodically. I hope it gets implemented soon.

Jeremy

**Comment by lukaszsocha2** [comment]
Hi @jeremyjong,
the recent release of `box-python-sdk-gen` supports setting a proxy. Please try it out [(docs)](https://github.com/box/box-python-sdk-gen/blob/main/docs/client.md#use-proxy-for-api-calls). If you have any problems using proxy, please open a new issue. I'm closing this one as your request was addressed.
Best,
@lukaszsocha2

**Comment by lukaszsocha2** [comment]
@jeremyjong it is now available in v1.5.0
